docs/user_manual/source/control_status_registers.rst

```diff
@@ -1611,8 +1611,22 @@ Detailed:
 +-------------+-----------+------------------------------------------------------------------------+
 |   Bit #     |  R/W      |   Description                                                          |
 +=============+===========+========================================================================+
-| 31:0        | R         | Machine Implementation ID **mimpid_i**, see  :ref:`core-integration`   |
+| 31:20       | R (0x)    | Hardwired to 0.                                                        |
 +-------------+-----------+------------------------------------------------------------------------+
+| 19:16       | R (0x)    | **MAJOR**.                                                             |
```

Comment:
Same as above, should this be "R (0x0)" ?

Reply:
Should be fixed now